NEWS

```diff
@@ -117,3 +117,4 @@ Changes in version 1.13.x
 + Added new functions getMediation and addMediation
 + replace getExperiment* and testExperiment* functions with getCrossAssociation
 + Replace mergeRows and mergeCols with new function agglomerateByVariable
++ use rbiom unifrac implementation instead of runUnifrac in calculateUnifrac file
```

Comment:
This is now in wrong line. You could also modify this to "Use rbiom package in unifrac implemenraton" or something